Shared/SentryIntegration.swift

```diff
@@ -90,6 +90,12 @@ public class SentryIntegration: SentryProtocol {
 
                 return event
             }
+            options.beforeBreadcrumb = { crumb in
+                if crumb.type == "http" || crumb.category == "http" {
```

Comment:
It's probably overkill to check both but the Sentry documentation is almost non-existent on this feature so I had to figure it out based on context and I can't be sure there are no cases where one is set but the other is not.